### PR TITLE
Zephyr: Disable the optional connection parameter update when testing L2CAP credict

### PR DESCRIPTION
**Pull request by ryanjh**

### CODE REVIEW

File: bot/config.py.zephyr.sample [hunk 1]
```diff
@@ -128,6 +128,11 @@ z['iut_config'] = {
                        'GAP/CONN/CPUP/BV-03-C', 'L2CAP/LE/CPU/BV-01-C']
     },
 
+    "l2cap_param_update.conf": {
```
Comment (kromancer) [inline bot/config.py.zephyr.sample]: can we call it `gap_auto_update_conn_params`?

there is already a config above with the same name

**Review by kromancer — CHANGES_REQUESTED** [review]

File: bot/config.py.zephyr.sample (hunk 1 above)
Comment (kromancer) [reply bot/config.py.zephyr.sample]: can you also check if we can merge these 2 configs (existing l2cap_param_update.conf with this new one)?

**Review by kromancer — APPROVED** [review]

File: bot/config.py.zephyr.sample (hunk 1 above)
Comment (ryanjh) [reply bot/config.py.zephyr.sample]: I gave it try merging gap_auto_update_conn_params.conf into l2cap_param_update.conf as below. However, the tests of 2cap_param_update.conf become INCONC. It appears to me, they require GAP_AUTO_UPDATE_CONN_PARAMS.

e.g.
1/4   GAP     GAP/CONN/CPUP/BV-01-C   INCONC          317.681        REGRESSION
2/4   GAP     GAP/CONN/CPUP/BV-02-C   INCONC          317.337        REGRESSION
3/4   GAP     GAP/CONN/CPUP/BV-03-C   INCONC          317.173        REGRESSION
4/4   L2CAP   L2CAP/LE/CPU/BV-01-C    INCONC          224.959        REGRESSION

z['iut_config'] = {
    "l2cap_param_update.conf": {
        "overlay": {'CONFIG_BT_CTLR_CONN_PARAM_REQ': 'n',
                    'CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS': 'n'},
        "test_cases": ['GAP/CONN/CPUP/BV-01-C', 'GAP/CONN/CPUP/BV-02-C',
                'GAP/CONN/CPUP/BV-03-C', 'L2CAP/LE/CPU/BV-01-C',
                 'L2CAP/LE/CFC/BV-06-C', 'L2CAP/LE/CFC/BV-18-C',
                 'L2CAP/LE/CFC/BV-19-C', 'L2CAP/LE/CFC/BV-21-C']},
}

Comment (kromancer) [reply bot/config.py.zephyr.sample]: Let me try this.

Are you suggesting that we should merge these 2 configurations?

Comment (ryanjh) [reply bot/config.py.zephyr.sample]: I prefer to separate them. To me, it is more readable that BV-06-C, BV-18-C, BV-19-C and BV-21-C only require AUTO_UPDATE_CONN_PARAMS overlay to make them pass.

**Pull request merged**